lib/bootstrap-bookingsync-sass.rb

```diff
@@ -10,9 +11,16 @@ def load!
           register_rails_engine
         elsif sprockets?
           register_sprockets
+        elsif defined?(::Sass) && ::Sass.respond_to?(:load_paths)
+          # The deprecated `sass` gem:
+          ::Sass.load_paths << stylesheets_path
         end
 
-        configure_sass
+        if defined?(::Sass::Script::Value::Number)
+          # Set precision to 6 as per:
+          # https://github.com/twbs/bootstrap/blob/da717b03e6e72d7a61c007acb9223b9626ae5ee5/package.json#L28
+          ::Sass::Script::Number.precision = [6, ::Sass::Script::Number.precision].max
```

Comment:
and here we could switch to:
https://github.com/sass/sassc-ruby/blob/61a02d1d88b489f83b5b3d0da48a9b62f210691b/lib/sassc/script/value/number.rb#L48

Reply:
Take from `bootstrap-sass` and seems legit for Sass users with Sprockets 3